test/game_statistic_test.rb

```diff
@@ -6,20 +6,24 @@
 require 'pry'
 
 
-class GameStatisticsTest < Minitest::Test
-  def test_highest_total_score
+class StatTrackerTest < Minitest::Test
+  def setup
     game_path = './data/dummy_game.csv'
-    team_path = './data/dummy_teams.csv'
+    team_path = './data/team_info.csv'
```

Comment:
Let's get together and set new dummy tests.

Reply:
Agree